test/integration/signature_editor_spec.mjs

```diff
@@ -671,4 +671,48 @@ describe("Signature Editor", () => {
       );
     });
   });
+
+  describe("Bug 1974257", () => {
+    let pages;
+
+    beforeEach(async () => {
+      pages = await loadAndWait("empty.pdf", ".annotationEditorLayer");
+    });
+
+    afterEach(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that the error panel is correctly removed", async () => {
```

Comment:
It looks like this is copy/pasted from another test. My suggestion would be to use `must check that the signature save checkbox is disabled if storage is full`.